src/inmanta/deploy/scheduler.py

```diff
@@ -169,16 +169,36 @@ async def deploy(self, priority: TaskPriority = TaskPriority.USER_DEPLOY) -> Non
         """
         Trigger a deploy
         """
+        match priority:
+            case TaskPriority.USER_DEPLOY:
+                reason = "User has requested a deploy"
+            case TaskPriority.INTERVAL_DEPLOY:
+                reason = "Deploy is being triggered by interval"
+            case _ as e:
+                raise RuntimeError(f"Unhandled case: {e}!")
```

Comment:
It would be good to look at the reasons we reported on the old agent and start from that.

Reply:
yes, I would put this on slack

Reply:
Comments on this one:

I would make all things concrete:

e.g.
`Deploy was triggered by timer` -> `Deploy start by timer at %s`
`A new version has been released` -> `Version %d has been released`